packages/api/src/functions/fixtures/apiGatewayProxyEvent.fixture.ts

```diff
@@ -22,6 +22,7 @@ export const mockedAPIGatewayProxyEvent: APIGatewayProxyEvent = {
       apiKey: null,
       apiKeyId: null,
       caller: null,
+      clientCert: null,
```

Comment:
@dac09 It seems the `@types/aws-lambda` package required this and was failing during build.

If CI passes, I'm going to merge. But please review this and update as needed in future PR.